### ISSUE TITLE
[Quest] Azuremyst Isle: Totem of Vark / Shadow of the Forest multiple casts

### ISSUE THREAD

**Issue by Ikarusit** [chromiecraft/chromiecraft#7415]
### What client do you play on?

enGB

### Faction

Alliance

### Content Phase:

1-19

### Current Behaviour

"Shadow of the Forest" cast gets triggered immediately on quest pickup, blocking you from seeing Yor spawn. It should only be triggered once when you followed Yor out of the water.

Quest: Totem of Vark [9542]
Creatures that are affected: Stillpine Ancestor Yor [17393]

### Expected Blizzlike Behaviour

After accepting the quest Totem of Vark [9542], Stillpine Ancestor Yor [17393] spawns, talks, walks with you out of the water, talks again and then "Shadow of the Forest" should get casted once on the Player, and the quest continues by a follow sequence.

Tldr.:

You should not get the Shadow of the Forest Buff on you when accepting the quest but just after Yor morphes himself.
Check this Video to get the correct order of this: https://youtu.be/G4JAWXTJ-vI

### Source

Playing through the quest on Classic and:
https://youtu.be/G4JAWXTJ-vI

### Steps to reproduce the problem

1. go to Azure Watch Azuremyst Isle
2. Advance questline (Start: "Learning the language" [9538], or skip this if you can use commands)
3. Go to the questgiver: Totem of Yor [creatureID: 17363]
4. Accept the quest named "Totem of Vark" [9542]
5. see that you immediately get "Shadow of the Forest" cast on you

### Extra Notes

Clients used for testing:
enUS + deDE

Weirdly enough, if you immediately cancel the buff "Shadow of the Forest" recieved by accepting the quest, it plays out as it should, and you can see Yor spawn in. He will even cast Shadow of the Forest at the right time on you again.
I looked through the SmartAI entrys for both the Totem of Yor and Stillpine Ancestor Yor on a fresh copy of azerothcores db, couldn't find anything wrong with them though.
Maybe someone else knows why it gets cast 2 times, I'm missing something here 100%.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/0fa770d56788196bda608343dde4ceb452848f89

### Operating system

Ubuntu 22.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft
